lib/cdo/script_constants.rb

```diff
@@ -186,7 +186,7 @@ def self.assignable_info(course_or_script)
       # Would better be called something like assignable_name
       script_name: course_or_script[:name],
       category: first_category,
-      position: ScriptConstants.position_in_category(name, first_category),
+      position: ScriptConstants.position_in_category(course_or_script[:name], first_category),
```

Comment:
Why this change?

Reply:
For minecraft adventurer and designer, `name` is "Minecraft Adventurer" and "Minecraft Designer", whereas the script names are `mc` and `minecraft`. The script names are used in the category hash that specifies all the ordering, so until now they've been incorrectly appearing at the bottom of the hoc category.

I added  a unit test for that particular bug.